src/Stratis.Bitcoin.Features.Consensus.Tests/Rules/CommonRules/PowCoinViewRuleTests.cs

```diff
@@ -0,0 +1,137 @@
+﻿using System;
+using System.Collections.Generic;
+using FluentAssertions;
+using Microsoft.Extensions.Logging;
+using Moq;
+using NBitcoin;
+using Stratis.Bitcoin.Base.Deployments;
+using Stratis.Bitcoin.Features.Consensus.CoinViews;
+using Stratis.Bitcoin.Features.Consensus.Rules;
+using Stratis.Bitcoin.Features.Consensus.Rules.CommonRules;
+using Stratis.Bitcoin.Utilities;
+using Xunit;
+
+namespace Stratis.Bitcoin.Features.Consensus.Tests.Rules.CommonRules
+{
+    /// <summary>
+    /// These tests only cover the first part of BIP68 and not the MaxSigOps, coinview update or scripts verify or calculate block rewards
```

Comment:
This is not really a new component, so adding any kind of tests should be sufficient as it is improvements to previous state. Full coverage might be too huge to include in one PR.

Reply:
Also it was really hard to get in there and test, but if we break this down into sub components it will get much easier to get tests around it. The tests I added were originally for the first sub rule when I had it broken down that way, but when the decision was made to defer that change, I put it back around this rule as a whole but obviously it didn't cover the whole rule, just only the first part. 

Ideally I wanted to cover all, it was just a big task to retro fit so we opted for line by line matching the new code to old and relying on manual testing and the review for the remaining parts.  Otherwise, I would have loved to have got the coverage.